src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -122,6 +125,43 @@ protected void loadConfiguration() {
     }
   }
 
+  private Timer pollTimer;
+
+  protected void stopPolling() {
```

Comment:
protected stat/stop polling methods to be called by the sub-classing `EppoClient` implementations in the java server and android SDKs.